src/components/Excursion/EditExcursion.js

```diff
@@ -0,0 +1,101 @@
+import React, {useState} from 'react';
+import PopUp from '../Cards/PopUp';
+import API from '../API/API'; 
+import LongInput from '../Cards/LongInput';
+
+function EditExcursion(props) {
+
+    /**
+     * @public EditExcursion renders the PopUp for editing excursions. Is a child of the component PopUp.s
+     * 
+     * TODO
+     * - Change date input to short inputs once fixed
+     * - Change description to big box
+    */
+    
+    const [excursionTitle, setExcursionTitle] = useState();
+    const [dateFrom, setDateFrom] = useState();
+    const [dateTo, setDateTo] = useState();
+    const [location, setLocation] = useState();
+    const [description, setDescription] = useState();
+
+    //const excursionData = [excursionTitle, dateFrom, dateTo, location, description];
+
+    function changeExcursionTitle(e) {
+        e.preventDefault();
+        setExcursionTitle(e.target.value);
+    }
+    function changeDateFrom(e) {
+        e.preventDefault();
+        setDateFrom(e.target.value);
+    }
+
+    function changeDateTo(e) {
+        e.preventDefault();
+        setDateTo(e.target.value);
+    }
+
+    function changeLocation(e) {
+        e.preventDefault();
+        setLocation(e.target.value);
+    }
+
+    function changeDescription(e) {
+        e.preventDefault();
+        setDescription(e.target.value);
+    }
+
+    const excursionID = null; //Add data source
+
+    /*The if-else statement determines whether data is currently there, 
+    meaning it should overwrite data or add data */
+    function handleSubmit(e) {
+        if (excursionID === null) { 
+            e.preventDefault();
+
+            if (!excursionTitle){
+                setExcursionTitle(e.target.value = "missing excursion title")
```

Comment:
is this changing the actual value of the input? it's a bit unusual but you can argue that it works because the user didn't input anything there...